tests/testTask.py

```diff
@@ -29,46 +29,53 @@
 import lsst.pex.config as pexConfig
 import lsst.pipe.base as pipeBase
 
+
 class AddConfig(pexConfig.Config):
     addend = pexConfig.Field(doc="amount to add", dtype=float, default=3.1)
 
+
 class AddTask(pipeBase.Task):
     ConfigClass = AddConfig
 
     @pipeBase.timeMethod
     def run(self, val):
         self.metadata.add("add", self.config.addend)
         return pipeBase.Struct(
-            val = val + self.config.addend,
+            val=val + self.config.addend,
```

Comment:
I'm sorry. I was wrong. I had not realised these were actually keyword arguments to a function. In which case the space around the equals is not allowed :-(